OpenSim/Simulation/OrientationsReference.h

```diff
@@ -116,14 +116,14 @@ class OSIMSIMULATION_API OrientationsReference
     int getNumRefs() const override;
     /** get the time range for which the OrientationsReference values are valid,
         based on the loaded orientation data.*/
-    SimTK::Vec2 getValidTimeRange() const override;
+    virtual SimTK::Vec2 getValidTimeRange() const override;
```

Comment:
DBC#2: `override` implies `virtual` - typical practice is to `virtual` the base class's method (it has nothing to override) and then `override` in all deriving classes

Reply:
Ah, okay. There were a few methods in this class that did not follow this pattern like the `hasNext()` method that has the `virtual` keyword in the `Reference` class, the `OrientationsReference` class, and the `BufferedOrientationsReference` class. I can revert these changes though. 

https://github.com/opensim-org/opensim-core/blob/849a8fbb58ecb75d3a400e6de27265f1094d6d0e/OpenSim/Simulation/Reference.h#L118

https://github.com/opensim-org/opensim-core/blob/849a8fbb58ecb75d3a400e6de27265f1094d6d0e/OpenSim/Simulation/OrientationsReference.h#L135

https://github.com/opensim-org/opensim-core/blob/849a8fbb58ecb75d3a400e6de27265f1094d6d0e/OpenSim/Simulation/BufferedOrientationsReference.h#L87